### PR TITLE
Change master to main

### PR DESCRIPTION
**Pull request by klaudiagrz**
**Description**

SAP is replacing insensitive terms (such as `master` / `slave`, `whitelist` / `blacklist`) with an inclusive language that helps to create a more inclusive workplace and ecosystem. To comply with this vision, we must adjust our code and documentation and correct the usage of these terms.

Changes proposed in this pull request:

- Rename `master` references to `main`, if possible

**Related issue(s)**
https://github.com/kyma-project/community/issues/515

### CODE REVIEW

File: components/provisioner/internal/installation/release/gcs_downloader.go [hunk 1]
```diff
@@ -48,10 +48,10 @@ func (o *GCSDownloader) DownloadRelease(version string) (model.Release, error) {
 
 // Detection rules:
 //   For pull requests: PR-<number>
-//   For changes to the master branch: master-<commit_sha>
-//   For the latest changes in the master branch: master
+//   For changes to the main branch: main-<commit_sha>
```
Comment (jasiu001) [inline components/provisioner/internal/installation/release/gcs_downloader.go:51]: it should no be changed, description is different than code

**Review by jasiu001 — APPROVED** [review]

**Review by kazydek — DISMISSED** [review]

**Review by jasiu001 — APPROVED** [review]

**Review by franpog859 — COMMENTED** [review]
I'm not sure if this won't break anything. According to [this doc](https://github.com/kyma-project/test-infra/blob/main/docs/prow/prow-architecture.md#generate-development-artifacts) the artifacts related to the `main` branch are still named like `master-...`. I'm going offline right now so I'm not able to test it but please double-check that everything will be ok with the "OnDemandRelease" providers/ downloaders

**Review by kazydek — APPROVED** [review]

**Review by franpog859 — APPROVED** [review]

**Review by kazydek — APPROVED** [review]

**Review by jasiu001 — APPROVED** [review]

**Pull request merged**